README.md

```diff
@@ -3,19 +3,17 @@
+[Bit Project](https://www.bitproject.org/) is a nonprofit organization that helps students find ways to compete in the tech industry by providing them countless tech educational resources. We offer a free open source alternative called "Bit Camp," created by student developers in collaboration with industry partners, such as Microsoft, Twilio, Slack, etc. The goal is to teach students how to become more proficient in learning programming concepts and applying them to the projects.
 
-Top schools provide their students with access to countless educational resources, student organizations, and networking opportunities outside of the classroom. Additionally, the “brand” of these schools alone can open doors for their graduates. But what about the majority of schools that don’t have these resources? Their students are left trying to find their own ways to stay marketable in a competitive industry. With their heavy course loads and busy schedules, students are not easily able to supplement their education by themselves. Non-traditional paths, like software engineering bootcamps, are often expensive and inaccessible to underserved communities. To help bridge this gap, we offer a free open source alternative, called “Bit Camps”, to teach beginner students a variety of topics relevant to the software engineering field. These programs can serve as a jumping off point for students who want to improve their tech skills or make their own side projects, but lack the appropriate resources or don't know where to begin. At their core, our programs are intended for anyone who is eager to learn new technologies. Whether you are already a tech student looking to improve your skills or someone who has never programmed a day in your life, we welcome you to join a Bit Camp and start learning! And if you are already confident in your skills, we are always eager for new contributors to help build our program curriculum!  
 
-## Bit Camps Repository
+## Introducing Bit Camps
 
-This repository includes folders of curriculum for each of our Bit Camps. Our Bit Camps consist of weekly livestreams and homework assignments and conclude with a final project. Student volunteers act as livestream hosts and TAs to teach the curriculum and conduct office hours. For those of our Bit Camps sponsored by industry partners, a developer advocate often co-hosts the livestream and engineers serve as mentors to help students understand concepts and complete their final projects. Although mentorship and feedback on homeworks/final projects are only available to students who apply and get into the bootcamp, the curriculum is free and open to all.
+Bit Camp consists of weekly live lectures (hosted by student developers), assignments, office hours, and the final project. Although mentorship and feedback are only available to students who apply and get into the boot camp, the curriculum is FREE and OPEN to all who want to learn new technologies and use their skills to their side projects. 
```

Comment:
Bit Camp consists of weekly live lectures (hosted by student developers), assignments, office hours, and **guidelines for creating a** final project. Although mentorship and feedback are only available to students who apply and get into the boot camp, the curriculum is FREE and OPEN to all who want to learn new technologies and use their skills to their side projects.